packages/web/docs/src/components/ecosystem-management/index.tsx

```diff
@@ -6,11 +6,13 @@ import {
   Heading,
   HighlightDecoration,
 } from '@theguild/components';
-import { cn } from '../lib';
-import { BookIcon } from './book-icon';
-import { CheckIcon } from './check-icon';
+import { cn } from '../../lib';
+import { BookIcon } from '../book-icon';
+import { CheckIcon } from '../check-icon';
 import styles from './ecosystem-management.module.css';
 
+const svgHref = new URL('./ecosystem-management.svg', import.meta.url).toString();
```

Comment:
`new URL` moves the responsibility of remembering about `basePath` to the bundler akin to Webpack's `file-loader`

`public` is nice and works everywhere, but it requires careful handling 😓